test/test_emulation/test_steelseries.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+
+from test_emulation import TestDevice, MouseData, MM_TO_INCH
+
+
+class TestSteelseriesDevice2(TestDevice):
+    def test_create(self, client):
+        id = client.create('steelseries-rival310')
+        client.delete(id)
+
+    def test_dpi(self, client, dpi_id=0):
+        id = client.create('steelseries-rival310')
+
+        old_dpi = 500
+        new_dpi = 1000
+
+        client.set_dpi(id, dpi_id, old_dpi)
+        subprocess.run(f'ratbagctl ratbag-emu resolution {str(dpi_id)} dpi set {str(new_dpi)}', shell=True)
```

Comment:
Also, this calls `ratbagctl` with `ratbag-emu` as device parameter. This works OK in the case you have only one test running, but will break when several are running in parallel.

A solution could be to generate a random device name and use it there instead.